r/R/get_expression_cellsets.R

```diff
@@ -0,0 +1,77 @@
+#' Get expression genes for pathway analysis
```

Comment:
this description does not seem correct. 

```suggestion
#' Get IDs for a new cell set that matches expression filters.
```